signalfx-java/src/main/java/com/signalfx/metrics/connection/AbstractHttpDataPointProtobufReceiverConnection.java

```diff
@@ -31,8 +31,9 @@ public abstract class AbstractHttpDataPointProtobufReceiverConnection extends Ab
 
     public AbstractHttpDataPointProtobufReceiverConnection(SignalFxReceiverEndpoint endpoint,
                                                            int timeoutMs,
+                                                           int maxRetries,
                                                            HttpClientConnectionManager httpClientConnectionManager) {
```

Comment:
Same comment about overloading constructor and maintaining backwards compatibility.